### DISCUSSION TITLE
External blog contributions

### DISCUSSION THREAD

**Discussion by pkafei** [vercel/vercel#6800] [Ideas] [answered]
Do you take blog posts ideas from the community? I would love to write an article on why technical writers and content creators should try Next.js. 

Many writers are encouraged to stick with platforms like Jekyll and Hugo. These platforms are considered industry standards for technical writers for good reasons- they are easy to use and edit content.  However, it is essential for writers interested in coding to consider a more robust and versatile tool than just a markdown generator. Next.js provides writers out-of-the-box server-side rendering and great technical SEO. 

If this topic sounds interesting, I'll submit a complete outline.

**Comment by leerob** [comment, marked as answer]
Hey! We're not currently accepting guest writers on the Vercel blog, but I'd love to read this on a platform like Hashnode!

https://hashnode.com/

**Reply by pkafei** [reply]
Thanks, @leerob for the prompt reply! Are there any ways for technical writers to contribute to Next.js? I don't see any tickets that are related to documentation or content.

**Reply by leerob** [reply]
There's not much on this repo, but I would look through the Next.js repo: https://github.com/vercel/next.js/issues

**Reply by pkafei** [reply]
Thank you!
